app/src/main/java/com/iic/shopingo/ui/OnboardingActivity.java

```diff
@@ -0,0 +1,110 @@
+package com.iic.shopingo.ui;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.ActionBarActivity;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import butterknife.InjectView;
+import com.facebook.Session;
+import com.facebook.SessionState;
+import com.facebook.UiLifecycleHelper;
+import com.facebook.widget.LoginButton;
+import com.iic.shopingo.R;
+
+public class OnboardingActivity extends ActionBarActivity {
```

Comment:
I don't think you want an actionbar in the login screen.
It should probably look more like a splashscreen.